app/js/sendData.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Send the new/updated JSON data to the database API
+ *
+ * @param {JSON} data JSON data to be sent
```

Comment:
incorrect, the data must be in FormData format at this point by using the JSONToFormData function